pkgs/applications/misc/img2pdf/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonApplication, fetchPypi, python3Packages }:
```

Comment:
`{ stdenv, python3Packages }` is sufficient. You can access the others as `python3Packages.fetchPypi` and `python3Packages.buildPythonApplication`.